wolfcrypt/src/aes.c

```diff
@@ -12769,6 +12903,131 @@ int wc_AesXtsEncrypt(XtsAes* xaes, byte* out, const byte* in, word32 sz,
     return ret;
 }
 
+#ifdef WOLFSSL_AESXTS_STREAM
+
```

Comment:
nitpick, but comment for `wc_AesXtsEncryptInit` would be nice.

Reply:
good catch.